packages/vaadin-app-layout/src/vaadin-drawer-toggle.js

```diff
@@ -84,12 +84,6 @@ class DrawerToggleElement extends Button {
         this.__fireClick();
       }
     });
-
```

Comment:
I suppose the `click` listener above could be simplified to not check for `disabled`. Not a blocker.

Reply:
Done.